doc/man3/SSL_CTX_set_tlsext_status_cb.pod

```diff
@@ -95,13 +118,17 @@ occurred).
 SSL_CTX_set_tlsext_status_cb(), SSL_CTX_set_tlsext_status_arg(),
 SSL_CTX_set_tlsext_status_type(), SSL_set_tlsext_status_type() and
 SSL_set_tlsext_status_ocsp_resp() return 0 on error or 1 on success.
+SSL_set_tlsext_status_ocsp_resp_ex() return 0 on error or 1 on success.
```

Comment:
Nit: either `returns` or just add it to the previous sentence.

Reply:
Added it to the sentence before.

Reply:
I do not see the fix.